web/src/components/__tests__/DatePicker.spec.tsx

```diff
@@ -1,25 +1,37 @@
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider'
 import { fireEvent } from '@testing-library/react'
 import { DateTime } from 'luxon'
 import React from 'react'
 
+import useCityContentParams from '../../hooks/useCityContentParams'
 import { renderWithTheme } from '../../testing/render'
-import DatePicker, { CustomDatePickerProps } from '../DatePicker'
+import DatePicker, { CustomAdapterLuxon, CustomDatePickerProps } from '../DatePicker'
 
 jest.mock('react-i18next')
+jest.mock('../../hooks/useCityContentParams')
 
+const mockedUseCityContentParams = useCityContentParams as jest.MockedFunction<typeof useCityContentParams>
 describe('DatePicker', () => {
   const setDate = jest.fn()
+  beforeEach(() => {
+    mockedUseCityContentParams.mockReturnValue({
+      cityCode: 'city',
+      languageCode: 'de',
+    })
+  })
```

Comment:
🙃 Instead of mocking this, you could also use `renderRoute` instead of `renderWithTheme` here with a city content route (e.g. `/augsburg/de/events`). Compare e.g. `ChatContainer.spec.tsx`.